monix-execution/shared/src/main/scala/monix/execution/internal/Trampoline.scala

```diff
@@ -89,3 +83,12 @@ private[execution] class Trampoline {
       }
     }
 }
+
+object Trampoline {
+  final class ResumeRun(head: Runnable, rest: ChunkedArrayQueue[Runnable], ec: ExecutionContext) extends Runnable {
+
+    def run(): Unit = new Trampoline(rest).immediateLoop(head, ec)
```

Comment:
Comment why creating new Trampoline instance is OK and why does not to be specialized for JVM (monix.execution.schedulers.TrampolineExecutionContext.JVMNormalTrampoline / monix.execution.schedulers.TrampolineExecutionContext.JVMOptimalTrampoline)